client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-expanded-content/style.scss

```diff
@@ -137,4 +142,80 @@ button.site-expanded-content__card-button {
 			white-space: nowrap;
 		}
 	}
+
+	.site-expanded-content__card-content-column-mobile {
+		.site-expanded-content__device-icon {
+			inset-inline-start: 0.2rem !important;
+		}
+	}
+}
+
+.site-expanded-content__help-icon {
+	fill: var(--studio-gray-10);
+	position: relative;
+	inset-block-start: 0.25rem;
+	inset-inline-start: 0.5rem;
+
+	path {
+		stroke-width: 0.5px;
+		stroke: var(--studio-gray-10);
+	}
+}
+
+.site-expanded-content__device-score-container {
+	display: flex;
+
+	.site-expanded-content__card-content-column {
+		position: relative;
+		inset-block-start: -5px;
+
+		.site-expanded-content__device-icon {
+			position: relative;
+			inset-block-start: 0.5rem;
+			inset-inline-start: -0.1rem;
+		}
+
+		.site-expanded-content__device-score {
+			font-size: 0.875rem;
+			color: var(--studio-gray-80);
+			margin-inline-start: 4px;
```

Comment:
On mobile devices under 363 px, the `margin-inline-start` isn't necessary:

![Screenshot 2023-03-09 at 14 02 29](https://user-images.githubusercontent.com/2293077/224128396-b70368a6-90f0-4011-80ee-f373da387c94.png)

Reply:
Good catch 👀